src/features/encoders/avif/client/index.tsx

```diff
@@ -38,10 +38,19 @@ interface State {
   denoiseLevel: number;
   aqMode: number;
   tune: AVIFTune;
+  enableSharpDownsampling: boolean;
```

Comment:
`aqMode` (line 39) doesn't seem to be used in codecs/avif/enc/avif_enc.cpp. Please double check.

Reply:
Yep it's an obsolete parameter, removing it.